### PR TITLE
Update headers to latest legal-approved headers.

### PR DESCRIPTION
**Pull request by hershey5045**
Please note the following cases when reviewing this PR:
## Case 1 - Files in the External folder
There should be no changes here

## Case 2 - Files in AtomTressFX folder that contain the AMD License
We need to append this license that indicates we've modified AMD code (According to correspondence with @lawsonamzn and Campbell)
```
/*
 * Modifications Copyright (c) Contributors to the Open 3D Engine Project. 
 * For complete copyright and license terms please see the LICENSE at the root of this distribution.
 * 
 * SPDX-License-Identifier: (Apache-2.0 OR MIT) AND MIT
 *
 */
```

## Case 3 - Other files in AtomTressFX
Regular header found in existing files
```
/*
 * Copyright (c) Contributors to the Open 3D Engine Project. 
 * For complete copyright and license terms please see the LICENSE at the root of this distribution.
 * 
 * SPDX-License-Identifier: Apache-2.0 OR MIT
 *
 */
```

Note that AR will catch files that were not tagged at this point.

### CODE REVIEW

File: Gems/AtomTressFX/External/license.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
+#
```
Comment (Adi-Amazon) [inline Gems/AtomTressFX/External/license.txt:1]: Please contact Nick and involve me in this - I'm not sure that we can add this to an external file that is open source by another company and under different license.

File: Gems/AtomTressFX/Tools/Maya/TressFX_Exporter.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
-# Copyright (c) Contributors to the Open 3D Engine Project
-# 
+#
```
Comment (Adi-Amazon) [inline Gems/AtomTressFX/Tools/Maya/TressFX_Exporter.py:1]: Same here - this file should follow the AMD license and Nick should be involved

**Review by Adi-Amazon — CHANGES_REQUESTED** [review]
Very important change.
Per several of my remarks - I believe that any file that was marked by AMD license cannot have our open source license and cannot be changed this way.
Please add Nick to this review and CC me on the any exchange

File: Gems/AtomTressFX/External/license.txt (hunk 1 above)
Comment (hershey5045) [reply Gems/AtomTressFX/External/license.txt:1]: Do we plan to include files from the external folder?

File: Gems/AtomTressFX/External/CMakeLists.txt [hunk 3]
```diff
@@ -1,13 +1,2 @@
...
-# its licensors.
-#
-# For complete copyright and license terms please see the LICENSE at the root of this
-# distribution (the "License"). All use of this software is governed by the License,
-# or, if provided, by the license below or the license accompanying this file. Do not
-# remove or modify any license notices. This file is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#
-
 #add_subdirectory(Code)
```
Comment (lawsonamzn) [inline Gems/AtomTressFX/External/CMakeLists.txt:1]: Whats up with this file?  It appears to be empty / commented out?  Why not delete it?

**Review by lawsonamzn — APPROVED** [review]
It looks good to me.
We cannot _change_ the license on files by others but we can publish them again, with teh same license.  The AMD license is MIT.  publishing it as SPDX-License "(Apache-2.0 OR MIT) AND MIT" means you have to comply with both if you republish.

**Review by Adi-Amazon — APPROVED** [review]

**Pull request merged**